presto-orc/src/main/java/com/facebook/presto/orc/metadata/DwrfMetadataReader.java

```diff
@@ -589,6 +598,8 @@ private static StreamKind toStreamKind(DwrfProto.Stream.Kind kind)
                 return StreamKind.ROW_GROUP_DICTIONARY;
             case STRIDE_DICTIONARY_LENGTH:
                 return StreamKind.ROW_GROUP_DICTIONARY_LENGTH;
+            case BLOOM_FILTER_UTF8:
```

Comment:
BLOOM_FILTER_UTF8 is a part of the DWRF index cache, idk why the OrcReader/DwrfMetadataReader do not support it.

Reply:
because it was added later :)